site/viamguides/src/securibot/securibot.md

```diff
@@ -0,0 +1,318 @@
+1. Once the Pi is started, connect to it with SSH. From a command line terminal window, enter the following command. The text in <> should be replaced (including the < and > symbols themselves) with the user and hostname you configured when you set up your Pi.
+   ```bash
+   ssh <USERNAME>@<HOSTNAME>.local
+   ```
+1. If you are prompted “Are you sure you want to continue connecting?”, type “yes” and hit enter. Then, enter the password for your username. You should be greeted by a login message and a command prompt.
+   ![raspberry pi SSH login](assets/sshLogin.webp)
+1. Update your Raspberry Pi to ensure all the latest packages are installed
+   ```bash
+   sudo apt update
+   sudo apt upgrade
+   ```
+
+<!-- ------------------------ -->
+
+## Set up the hardware
+
+Duration: 5
+
+The Viam office currently uses a fob system to unlock a security door. By adding computer vision and facial recognition, we can verify that specific individuals are authorized to enter and automatically unlock the door. In this setup, a webcam is mounted above the door to detect faces. When an authorized face is recognized, a servo waves a heated pad in front of a PIR motion sensor (originally intended for people leaving the office) to trigger the door to unlock.
+
+Your door setup may differ, so this section may need to be adjusted based on your specific configuration.
+
+### Add your servo
+
+1. **Review the wiring diagram**: The servo can be controlled via a GPIO pin on the Raspberry Pi. Refer to the following wiring diagram to see how to connect the Raspberry Pi to the servo, using the breadboard and also resistors to control the flow of electricity. The breadboard simplifies the wiring process by providing a secure platform to connect components without soldering.
+
+   ![wiring diagram](assets/securibot_bb.png)
+
+   | **Raspberry Pi** | **Servo and Heated pad** |
+   | ---------------- | ------------------------ |
+   | Pin 2 (5V)       | Power for heated pad     |
+   | Pin 4 (5V)       | Power for servo          |
+   | Pin 6 (Ground)   | GND for servo            |
+   | Pin 9 (Ground)   | GND for heated pad       |
+   | Pin 16 (GPIO 23) | GPIO for servo           |
+
+   > aside positive
+   > The website [pinout.xyz](https://pinout.xyz/) is a helpful resource with the exact layout and role of each pin for Raspberry Pi. When working with Viam, make sure to reference the physical pin numbers, and not the GPIO numbers listed on `pinout.xyz`.
+
+1. **Wire the servo to the Raspberry Pi**: Connect the peripherals referencing the wiring diagram from above. You can optionally use a breadboard to simplify the wiring process by providing a secure platform to connect components without soldering.
```

Comment:
Is this instruction redundant? It seems like it's basically saying the same thing as the first instruction "Review the wiring diagram".